modules/core/04-channel/types/channel.go

```diff
@@ -75,6 +75,12 @@ func (ch Channel) ValidateBasic() error {
 	return ch.Counterparty.ValidateBasic()
 }
 
+// Close modifies the Channel with the new State set to CLOSED and FlushStatus set to NOTINFLUSH.
+func (ch *Channel) Close() {
+	ch.State = CLOSED
+	ch.FlushStatus = NOTINFLUSH
+}
```

Comment:
if we refactor things to put the flush status on the `Upgrade` then I don't think we'll need this method

Reply:
yep good point, We can maybe just revert this PR if that happens